Parsing.py

```diff
@@ -0,0 +1,22 @@
+def parsing(bed_file):
+    def int_error(our_int: str):
+        if our_int.isdigit() is False:
+            raise ValueError("{0} is not a integer".format(our_int))
+    def str_error(our_str: str):
+        if isinstance(our_str, str) is False:
+            raise ValueError("{0} is not a integer".format(our_str))
+    def checking(our_line: list):
+        str_error(our_line[0])
+        int_error(our_line[1])
+        int_error(our_line[2])
+        try:
```

Comment:
In this case error catching is not the best way. Error object creation is not cheap, so it will slow a program, if there are many error being thrown. And this is not an exceptional case, so just check length of the list.